src/main/resources/db/migration/V1__create_schema.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE users(
+    id       INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
Let's rename this to `user_id`. This would allow an unambiguous joining (even using `ON`) and it's generally recommended naming style.